src/main/java/com/salesforce/emp/connector/LoginHelper.java

```diff
@@ -77,7 +77,7 @@ public void startElement(String uri, String localName, String qName, Attributes
         }
     }
 
-    public static final String COMETD_REPLAY = "/cometd/replay/";
```

Comment:
This is incorrect unfortunately.  Do you have a case where this is not working for you?  The trailing "replay" was only needed in a previous version, handled below.  Is that not working correctly?